confgenerator/confgenerator_test.go

```diff
@@ -43,14 +43,8 @@ var (
 	//   ops-agent$ go test -mod=mod github.com/GoogleCloudPlatform/ops-agent/confgenerator -update_golden
 	// Add "-v" to show details for which files are updated with what:
 	//   ops-agent$ go test -mod=mod github.com/GoogleCloudPlatform/ops-agent/confgenerator -update_golden -v
-	updateGolden      = flag.Bool("update_golden", false, "Whether to update the expected golden confs if they differ from the actual generated confs.")
```

Comment:
They're sort of bundled since `modify_fields` adds golden Lua files to the tests. Deferring the refactor would mean either not including those tests here or doing a different/temporary refactor to accommodate the Lua files.

Reply:
Oh I meant to split it into a separate PR and get that one in first.